### DISCUSSION TITLE
Documentation improvement - show fn syntax at top of the page

### DISCUSSION THREAD

**Discussion by AndreiIgna** [js-fns/date-fns#2490] [General]
I use `date-fns` in many projects, and sometimes I need to check the documentation to see a function's syntax.

Every time I open a docs page (ex: https://date-fns.org/v2.16.1/docs/format) I was expecting to see the usage, arguments, etc at the top. But I see the descriptions are at the top, and need to scroll a long way to find this:
<img width="1075" alt="Screenshot 2021-05-25 at 15 38 51" src="https://user-images.githubusercontent.com/263021/119510756-11f32180-bd72-11eb-9a12-93435e68416b.png">
For this specific page, an important piece of information is almost at the end 😕

Other projects show usage or function name with accepted arguments at the top:
<img width="899" alt="Screenshot 2021-05-25 at 16 01 41" src="https://user-images.githubusercontent.com/263021/119512376-89758080-bd73-11eb-88a5-2b66accdc913.png">
<img width="636" alt="Screenshot 2021-05-25 at 16 02 06" src="https://user-images.githubusercontent.com/263021/119512390-8d090780-bd73-11eb-93e9-da245303ca8e.png">

and I think showing function syntax at the top will be an improvment for date-fns. Ex:
![Frame 3](https://user-images.githubusercontent.com/263021/119513325-69928c80-bd74-11eb-9524-3d325a1fe628.png)

Just wanna start a discussion to see if others noticed this and suggest an option. Sometimes the code editor with autocomplete is enough, but there are moments when we check the docs site too.

**Comment by tan75** [comment]
Hi @AndreiIgna 

Thank you for the idea - this is a great suggestion and a PR is welcome! The only thing I would suggest is to make the syntax look as follows: 

![image](https://user-images.githubusercontent.com/8555279/120006192-8cb97800-bfd0-11eb-9591-f44c6a495179.png)

Could you please submit a PR if you wish to continue?
